hudi-common/src/main/java/org/apache/hudi/common/table/view/FileSystemViewStorageConfig.java

```diff
@@ -86,6 +86,12 @@
   // Need to be disabled only for tests.
   public static final String DEFAULT_REMOTE_BACKUP_VIEW_HANDLER_ENABLE = "true";
 
+  /**
+   * Configs to control whether to make the embedded timeline service singleton.
+   */
+  public static final String FILESYSTEM_VIEW_SINGLETON = "hoodie.filesystem.view.singleton.enable";
```

Comment:
I think it's better to keep this config with the ones that enable the timeline server for e.g `hoodie.embed.timeline.server` is in `HoodieWriteConfig`.  Can we add a `hoodie.timeline.server.reuse.enabled` there?

Reply:
Agree.